static/js/settings_display.js

```diff
@@ -224,6 +227,23 @@ export function set_up(settings_panel) {
             },
         });
     });
+
+    $container.find(".setting_right_sidebar_style_value").on("click", function () {
+        const data = {right_sidebar_style: $(this).val()};
+        const current_right_sidebar_style = settings_object.right_sidebar_style;
+        if (current_right_sidebar_style === data.right_sidebar_style) {
+            return;
+        }
+
+        channel.patch({
+            url: "/json/settings",
+            data,
+            success() {},
+            error() {
+                // TODO: handle error here
```

Comment:
@timabbott what would be the correct way to handle the error path here? I've left it blank for now.

Reply:
The `ui_report.error` invocation currently used in the emoji block just above should be correct.